### ISSUE TITLE
Archive jobs

### ISSUE THREAD

**Issue by mesmith75** [ganga-devs/ganga#2090]
A request from a user:
It would be nice to be able to archive jobs such that they don't appear when `jobs` is typed unless they are requested

**Comment by alexanderrichards** [comment]
interesting idea. It might have some merit.

**Comment by egede** [comment]
Yes, we have briefly discussed this before. To avoid having to create new states of jobs and still having the jobs in the repository and thus slowing down things - I think the best would be to migrate the jobs to a new `gangadir` location. Then to access them, you would simply start `ganga` with that `gangadir` location. It should only be possible for jobs in a final state (`completed`, `failed`, `killed`) such that they are not touched by the monitoring.

This is a quite nice standalone issue

**Comment by AasmaGupta** [comment]
if this issue isnt resolved yet, id like to work on it!

**Comment by egede** [comment]
You are very welcome to do so. It will require
- Installing Ganga and making yourself familiar with how jobs are executed and how they are persisted between sessions.
- Understand how a job can be read from one `repository` and written into another. When Ganga starts, we already have a template repository in addition to the standard repository. The way that a template is read and then use to create a job is quite similar.
- Propose an API that the user would use to initiate a migration to an archive.
- Implement the API and create test cases.

**Comment by AasmaGupta** [comment]
Thanks a lot, I am starting to learn and work on it!
